File: ws-C++/hora.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){

    int hora;
    cout << "Digite uma hora do dia: ";
    cin >> hora;

    if(hora < 12){
        cout << "Bom dia!" << endl;
    }
    else if(hora < 18){
        cout << "Boa tarde!" << endl;
    }
    else{
        cout << "Boa noite!" << endl;
    }

    return 0;
}

File: ws- C#/Hora.cs
using System;

namespace Hora
{
    class Hora
    {
        static void Main(string[] args)
        {
            int hora;

            Console.Write("Digite uma hora do dia: ");
            hora = int.Parse(Console.ReadLine());

            if (hora < 12)
            {
                Console.WriteLine("Bom dia!");
            }
            else if (hora < 18)
            {
                Console.WriteLine("Boa tarde!");
            }
            else
            {
                Console.WriteLine("Boa noite!");
            }
        }
    }
}
